# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/gnomesword/glib-h-fix.patch
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -urN gnomesword2-2.0.0/src/backend/config.cc gnomesword2-2.0.0-my/src/backend/config.cc
--- gnomesword2-2.0.0/src/backend/config.cc	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/backend/config.cc	2004-01-09 04:21:54.000000000 +0100
@@ -44,7 +44,7 @@
 #include <sys/stat.h>
  
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/backend/key.cc gnomesword2-2.0.0-my/src/backend/key.cc
--- gnomesword2-2.0.0/src/backend/key.cc	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/backend/key.cc	2004-01-09 04:21:54.000000000 +0100
@@ -32,7 +32,7 @@
 
 #include <swmodule.h>
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/backend/sword.cpp gnomesword2-2.0.0-my/src/backend/sword.cpp
--- gnomesword2-2.0.0/src/backend/sword.cpp	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/backend/sword.cpp	2004-01-09 04:21:54.000000000 +0100
@@ -31,7 +31,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/gnome2/toolbar_nav.c gnomesword2-2.0.0-my/src/gnome2/toolbar_nav.c
--- gnomesword2-2.0.0/src/gnome2/toolbar_nav.c	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/gnome2/toolbar_nav.c	2004-01-09 04:21:55.000000000 +0100
@@ -24,7 +24,7 @@
 #endif
 
 #include <gnome.h>
-#include <gal-2.0/gal/widgets/e-unicode.h>
+#include <gal/widgets/e-unicode.h>
 
 #include "gui/toolbar_nav.h"
 #include "gui/shortcutbar_main.h"
diff -urN gnomesword2-2.0.0/src/main/bibletext.c gnomesword2-2.0.0-my/src/main/bibletext.c
--- gnomesword2-2.0.0/src/main/bibletext.c	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/bibletext.c	2004-01-09 04:21:54.000000000 +0100
@@ -24,7 +24,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/main/commentary.c gnomesword2-2.0.0-my/src/main/commentary.c
--- gnomesword2-2.0.0/src/main/commentary.c	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/commentary.c	2004-01-09 04:21:54.000000000 +0100
@@ -24,7 +24,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/main/configs.c gnomesword2-2.0.0-my/src/main/configs.c
--- gnomesword2-2.0.0/src/main/configs.c	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/configs.c	2004-01-09 04:21:54.000000000 +0100
@@ -24,7 +24,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/main/configs.h gnomesword2-2.0.0-my/src/main/configs.h
--- gnomesword2-2.0.0/src/main/configs.h	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/configs.h	2004-01-09 04:21:54.000000000 +0100
@@ -26,7 +26,7 @@
 extern "C" {
 #endif	
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/main/dictlex.c gnomesword2-2.0.0-my/src/main/dictlex.c
--- gnomesword2-2.0.0/src/main/dictlex.c	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/dictlex.c	2004-01-09 04:21:54.000000000 +0100
@@ -24,7 +24,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/main/gbs.c gnomesword2-2.0.0-my/src/main/gbs.c
--- gnomesword2-2.0.0/src/main/gbs.c	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/gbs.c	2004-01-09 04:21:54.000000000 +0100
@@ -28,7 +28,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/main/key.c gnomesword2-2.0.0-my/src/main/key.c
--- gnomesword2-2.0.0/src/main/key.c	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/key.c	2004-01-09 04:21:54.000000000 +0100
@@ -23,7 +23,7 @@
 #include <config.h>
 #endif
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/main/lists.c gnomesword2-2.0.0-my/src/main/lists.c
--- gnomesword2-2.0.0/src/main/lists.c	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/lists.c	2004-01-09 04:21:54.000000000 +0100
@@ -24,8 +24,8 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
-#include <gal-2.0/gal/widgets/e-unicode.h>
+#include <glib.h>
+#include <gal/widgets/e-unicode.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/main/lists.h gnomesword2-2.0.0-my/src/main/lists.h
--- gnomesword2-2.0.0/src/main/lists.h	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/lists.h	2004-01-09 04:21:54.000000000 +0100
@@ -27,7 +27,7 @@
 {
 #endif
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/main/module.c gnomesword2-2.0.0-my/src/main/module.c
--- gnomesword2-2.0.0/src/main/module.c	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/module.c	2004-01-09 04:21:54.000000000 +0100
@@ -26,7 +26,7 @@
 
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/main/percomm.c gnomesword2-2.0.0-my/src/main/percomm.c
--- gnomesword2-2.0.0/src/main/percomm.c	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/percomm.c	2004-01-09 04:21:54.000000000 +0100
@@ -25,7 +25,7 @@
 
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/main/search.c gnomesword2-2.0.0-my/src/main/search.c
--- gnomesword2-2.0.0/src/main/search.c	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/search.c	2004-01-09 04:21:54.000000000 +0100
@@ -24,7 +24,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/main/search.h gnomesword2-2.0.0-my/src/main/search.h
--- gnomesword2-2.0.0/src/main/search.h	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/search.h	2004-01-09 04:21:54.000000000 +0100
@@ -27,7 +27,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/main/settings.c gnomesword2-2.0.0-my/src/main/settings.c
--- gnomesword2-2.0.0/src/main/settings.c	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/settings.c	2004-01-09 04:21:54.000000000 +0100
@@ -25,7 +25,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/main/shortcutbar.c gnomesword2-2.0.0-my/src/main/shortcutbar.c
--- gnomesword2-2.0.0/src/main/shortcutbar.c	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/shortcutbar.c	2004-01-09 04:21:54.000000000 +0100
@@ -24,7 +24,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/main/shortcutbar.h gnomesword2-2.0.0-my/src/main/shortcutbar.h
--- gnomesword2-2.0.0/src/main/shortcutbar.h	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/shortcutbar.h	2004-01-09 04:21:54.000000000 +0100
@@ -27,7 +27,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/main/spell_pspell.c gnomesword2-2.0.0-my/src/main/spell_pspell.c
--- gnomesword2-2.0.0/src/main/spell_pspell.c	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/spell_pspell.c	2004-01-09 04:21:54.000000000 +0100
@@ -26,7 +26,7 @@
 #ifdef USE_PSPELL
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/main/sword.c gnomesword2-2.0.0-my/src/main/sword.c
--- gnomesword2-2.0.0/src/main/sword.c	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/sword.c	2004-01-09 04:21:54.000000000 +0100
@@ -24,7 +24,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
diff -urN gnomesword2-2.0.0/src/main/xml.c gnomesword2-2.0.0-my/src/main/xml.c
--- gnomesword2-2.0.0/src/main/xml.c	2003-12-24 20:10:38.000000000 +0100
+++ gnomesword2-2.0.0-my/src/main/xml.c	2004-01-09 04:21:54.000000000 +0100
@@ -24,7 +24,7 @@
 #endif
 
 #ifdef USE_GNOME2
-#include <glib-2.0/glib.h>
+#include <glib.h>
 #else
 #include <glib-1.2/glib.h>
 #endif
